overlord/snapstate/snapstate_test.go

```diff
@@ -5300,93 +5301,94 @@ func (s contentStore) SnapAction(ctx context.Context, currentSnaps []*store.Curr
 	if err != nil {
 		return nil, nil, err
 	}
-	if len(sars) != 1 {
-		panic("expected to be queried for install of only one snap at a time")
-	}
```

Comment:
should this check be restored?

Reply:
Yeah that's fine, although it needs to be relaxed now and allow 1 or 2 actions. Updated, thanks.